samples/tfm_integration/tfm_psa_test/src/main.c

```diff
@@ -9,7 +9,6 @@
 /* Run the PSA test suite */
 void psa_test(void);
 
-__attribute__((noreturn))
 int main(void)
```

Comment:
Does this need to return something even with the for loop in the end? Some toolchains might complain that there is no return statement.

Reply:
You have the choice between a warning about a missing return and a warning for unreachable code. It looks like gcc (at least) is happy with the code as-is.

Reply:
@dcpleung Any further feedback on this one?

Reply:
No. As long as the compiler is happy, I am fine with this.